tests/90jira/SYN-627.pl

```diff
@@ -23,7 +25,9 @@
             my $room_id = $_;
 
             matrix_send_room_text_message( $user, $room_id, body => "$room_id" );
-         } @rooms );
+         } ( head -1, @rooms) );
```

Comment:
If this is the only use of `head` here, you could just use

    $rooms[-1]

to save an entire new CPAN import of a module. `head` is more intended for cases like slicing the result of another function call, or getting more than just the final element.